snuba/admin/iam_policy/iam_policy.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
this should probably live in the ops repo right?

Reply:
makes sense. I'll move it to ops.